src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/TranslatedDocumentUploadedDefendantNotificationHandlerTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.notification;
+
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.Party;
+import uk.gov.hmcts.reform.civil.notify.NotificationService;
+import uk.gov.hmcts.reform.civil.notify.NotificationsProperties;
+import uk.gov.hmcts.reform.civil.sampledata.CallbackParamsBuilder;
+import uk.gov.hmcts.reform.civil.sampledata.CaseDataBuilder;
+import uk.gov.hmcts.reform.civil.sampledata.PartyBuilder;
+
+import java.util.Map;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.notification.NotificationData.CLAIM_REFERENCE_NUMBER;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.notification.NotificationData.RESPONDENT_NAME;
+import static uk.gov.hmcts.reform.civil.utils.PartyUtils.getPartyNameBasedOnType;
+
+@SpringBootTest(classes = {
+    TranslatedDocumentUploadedDefendantNotificationHandler.class,
+    JacksonAutoConfiguration.class
+})
+public class TranslatedDocumentUploadedDefendantNotificationHandlerTest {
+
+    @MockBean
+    private NotificationService notificationService;
+    @MockBean
+    private NotificationsProperties notificationsProperties;
+    @Autowired
+    private TranslatedDocumentUploadedDefendantNotificationHandler handler;
+
+    @Nested
+    class AboutToSubmitCallback {
+
+        @BeforeEach
+        void setup() {
+            when(notificationsProperties.getNotifyDefendantTranslatedDocumentUploaded()).thenReturn("template-id");
+        }
+
+        @Test
+        void shouldNotifyApplicantParty_whenInvoked() {
+            Party party = PartyBuilder.builder()
+                .individual("respondent")
+                .partyEmail("respondent@example.com")
+                .build();
+
+            CaseData caseData = CaseDataBuilder.builder().atStateClaimDetailsNotified()
+                .setClaimTypeToSpecClaim()
+                .respondent1(party)
+                .build();
+            CallbackParams params = CallbackParamsBuilder.builder().of(ABOUT_TO_SUBMIT, caseData).request(
+                CallbackRequest.builder().eventId("NOTIFY_DEFENDANT_TRANSLATED_DOCUMENT_UPLOADED")
```

Comment:
same applies here 😄 sorry did not notice during my previous reviews

Reply:
done